BraintreeCore/src/main/java/com/braintreepayments/api/PostalAddress.java

```diff
@@ -4,6 +4,9 @@
 import android.os.Parcelable;
 import android.text.TextUtils;
 
+import androidx.annotation.NonNull;
```

Comment:
It looks like nothing is set as `@NonNull` in this file. Does this still need to be imported here?